src/openrct2-ui/windows/Finances.cpp

```diff
@@ -517,7 +525,7 @@ rct_window * window_finances_open()
 
     w = window_bring_to_front_by_class(WC_FINANCES);
     if (w == nullptr) {
-        w = window_create_auto_pos(530, 290, _windowFinancesPageEvents[0], WC_FINANCES, WF_10);
+        w = window_create_auto_pos(530, 310, _windowFinancesPageEvents[0], WC_FINANCES, WF_10);
```

Comment:
Would be nice to have these in defines, so there's only one place to change them when needed.

Reply:
Like a `WW` and `WH`? Trouble is, each of the tabs have different widths and heights, as you may recall. Properly refactoring that is beyond the scope of this PR, I think.